packages/devtools_app/lib/src/debugger/hover.dart

```diff
@@ -158,3 +174,84 @@ class HoverCard {
     _isRemoved = true;
   }
 }
+
+class HoverCardTip extends StatefulWidget {
```

Comment:
I don't understand the naming here. "tip" makes me think multiple things initially. Is it a hint? is it a tooltip? Is it describing the point at the bottom of the hover card?

Reply:
Changed name to HoverCardTooltip and added a comment.